projects/core/src/lib/lesson/lesson.component.scss

```diff
@@ -0,0 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400&&display=swap");
```

Comment:
maybe it is better way to put this import in styles.scss, so that the font is available to every component

Reply:
but if I add it to the general style.scss file then everyone's font will change, because they are different

Reply:
done